src/tools/openpgp-tool.c

```diff
@@ -479,14 +588,16 @@ int main(int argc, char **argv)
 	r = util_connect_card(ctx, &card, opt_reader, opt_wait, verbose);
 	if (r) {
 		util_fatal("failed to connect to card: %s\n",
-			sc_strerror(r));
+		           sc_strerror(r));
```

Comment:
Hi @frankmorgner 
I use tab for indentation. For the space, it is for keeping alignment with the brackets of the upper line. I use space for alignment because if I use tab, when someone set the editor tab with to 2 or 8, the alignment will be lost. Using spaces here will retain the distance no matter how long the tab is.

Can I keep the spaces?